packages/brokeneck-fastify/lib/pluginConfig.js

```diff
@@ -2,6 +2,7 @@
 
 const envSchema = require('env-schema')
 const S = require('fluent-json-schema')
+const Ajv = require('ajv')
```

Comment:
Note that we are using the fastify's ajv dependency (v6 by default).

We should add `ajv` to the dependancies

Reply:
My bad, I forgot to add the changes to the package.json, will I do a new PR with the changes?

Reply:
I agree with that